Sources/PublishCLICore/CLI.swift

```diff
@@ -63,10 +68,17 @@ private extension CLI {
 
         - new: Set up a new website in the current folder.
         - generate: Generate the website in the current folder.
-        - run: Generate and run a localhost server on port 8000
-               for the website in the current folder.
+        - run: Generate and run a localhost server on default port 8000
+               for the website in the current folder. Use the
```

Comment:
Think you could move the "-p" up here to make the text wrap a bit more neatly. Nitpick, I know 😅

Reply:
I understand that ;) I fixed it and it does look nicer ;)